src/modules/predefinedDimensions.js

```diff
@@ -10,6 +10,8 @@ export const DIMENSION_ID_ORGUNIT = 'ou'
 export const DIMENSION_ID_ASSIGNED_CATEGORIES = 'co'
 export const DIMENSION_PROP_NO_ITEMS = 'noItems'
 
+export const DIMENSION_TYPE_DATA_ELEMENT_GROUP_SET = 'DATA_ELEMENT_GROUP_SET'
```

Comment:
This should probably be placed somewhere else as this file only handles actual dimensions where as `DIMENSION_TYPE_DATA_ELEMENT_GROUP_SET` is a _dimension type_.

Reply:
https://github.com/dhis2/analytics/blob/7c8658f5b2da1611561616a1c75bd6a87d526c24/src/modules/layout/dimension.js ?
Or a new file in `src/modules` called `dimensionTypes.js`?

Reply:
It's right below another constant that's tangentially related to dimensions, but could move it to `pivotTableConstants` if this is important.

Reply:
@martinkrulltott @janhenrikoverland I still think this is OK here, let me know if I should move it to `pivotTableConstants`

Reply:
`predefinedDimensions.js` contains predefined dimensions. This is neither predefined nor a dimension. It's a dimension type. Should it be there? No. Can it still be there? Sure 🤷‍♂ 🙂

Reply:
We don't have constants for any other dynamic dimension types (orgunit group sets, categories etc), so `pivotTableConstants` makes sense for now I think.

Reply:
OK, done